Documentation/Upgrade/README.rst

```diff
@@ -0,0 +1,106 @@
+.. _README:
+
+.. include:: ../lib/doc-includes/pike-icons.inc
+
+.. Don't use default python highlighting for code blocks http://www.sphinx-doc.org/en/stable/markup/code.html
+
+.. highlight:: none
+
+
+===================================
+Minor Upgrade to Release 5.4.1 U8+
+===================================
+
+.. contents::
+   :local:
+   :depth: 3
+
+
+Use this documentation when upgrading between minor releases. The process applies to upgrades from Release 5.4.1 U5 to Release 5.4.1 U8+. During this process, the Nuage components and Red Hat can be updated at the same time.
+
+It is assumed the operator is familiar with OpenStack upgrades, VSP installation, the distribution-specific installation and upgrade practices, and the specific requirements for operations in a production environment.
+
+
+Upgrade Paths
+-------------
+
+Upgrading to the Release 5.4.1 U8+ is supported only on OSPD-based deployments. In this release, you can upgrade only from Queens 5.4.1 U5 to Queens 5.4.1 U8.
+    
+
+These upgrade paths are not described in this document:
+    
+    * Upgrade from OpenStack releases before Queens 5.4.1 U5
+    * Upgrade from VSP releases before Release 5.4.1 U5
+    
+
+If you are upgrading from a release earlier than Queens 5.4.1 U5, refer to the Nuage Queens upgrade documentation.
+
+
+Basic Configuration
+---------------------
+
+The basic configuration includes:
+   
+   * An existing installed VSD and VSC upgraded to latest VSP release
+   * A single OpenStack Controller node
+   * One or more Compute nodes (hypervisors) running the VRS with the latest VSP release
+   
+
+
+Before the Upgrade
+--------------------
+
+1. If you are upgrading an AVRS node, migrate all your VMs on the node to an AVRS Compute node that is not being upgraded. Perform the steps in the "Live Migrate a Virtual Machine" section in https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html/director_installation_and_usage/migrating-virtual-machines-between-compute-nodes-osp#live-migrate-a-vm-osp
+
+2. Create the following repositories:
+    
+    * Nuage_base_repo:
+        
+        - It contains Selinux-policy-nuage, nuage-bgp, Python-openvswitch-nuage, nuage-openstack-neutronclient, and nuage-puppet-modules.
+        - Enable this repository on the on OpenStack Controller nodes and on the Compute VRS or AVRS nodes.
+    
+    * Nuage_vrs_repo:
+        
+        - It contains nuage-metadata and nuage-openvswitch.
+        - Enable this repository on the OpenStack Controller node and the Nova Compute VRS nodes.
+        

```

Comment:
We don't need to mention it is nova compute, or we make it also mentioned for Nuage_base_repo.

Reply:
done - added to nuage_base_repo